packages/@react-spectrum/s2/src/index.ts

```diff
@@ -10,6 +10,8 @@
  * governing permissions and limitations under the License.
  */
 
+'use client';
```

Comment:
due to how our build works, this won't actually affect our released packages at the moment, so it'll only be used by the docs.